src/Likvido.DbMigrator.EfCore/MigrationsRunner.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Likvido.Extensions.Logging;
+using Likvido.Payables.Services.Data;
+using Likvido.Robots;
+using Microsoft.ApplicationInsights;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog;
+
+namespace Likvido.DbMigrator
+{
+    public class MigrationsRunner<TContext, TContextFactory>
+        where TContextFactory : IDesignTimeDbContextFactory<TContext>
+        where TContext: DbContext
+    {
+        private ILogger<MigrationsRunner<TContext, TContextFactory>>? _logger;
+        private readonly string _role = "payables-db-migrator";
```

Comment:
Don't think we should have a default role specified here

```suggestion
        private readonly string _role;
```